### PR TITLE
ci: ZAP API github action

### PR DESCRIPTION
**Pull request by theblockstalk**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -27,3 +27,30 @@ jobs:
...
+      - run: yarn install --immutable
+      - run: yarn run build
+      - run: yarn run start:prod &
+      - run: sleep 3
+
+      - name: ZAP Scan
+        uses: zaproxy/action-api-scan@v0.3.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          target: 'http://localhost:5000/openapi.json'
```
Comment (sadiabbasi) [inline .github/workflows/test.yaml:54]: can you explain this like how you link this target?

Comment (theblockstalk) [reply .github/workflows/test.yaml:54]: https://github.com/Tonomy-Foundation/Tonomy-Communication/pull/54/files#diff-769911c416ccf8514d8fd941ae0abe8fb5c606ade0c218e22151a5f5f9f3d700R15

At this point we register the OpenAPI document to be served at this path

**Pull request merged**